src/wx/wxvbam.cpp

```diff
@@ -1192,24 +1212,9 @@ void MainFrame::IdentifyRom()
 // a few keys (e.g. only ctrl-x works for exit, but not esc & ctrl-q;
 // ctrl-w does not work for close).  It's possible another entity is
 // grabbing those keys, but I can't track it down.
```

Comment:
This is not Windows only... On Linux, I was trying to setup the `L` (no modifier) to open files, but it was being ignored. It seems to be something related to `wxGTK`. No idea what was causing this on Windows.